pkg/operation/common/utils.go

```diff
@@ -506,3 +503,38 @@ func DeleteAlertmanager(k8sClient kubernetes.Interface, namespace string) error
 	}
 	return nil
 }
+
+// GetDomainInfoFromAnnotations returns the provider and the domain that is specified in the give annotations.
+func GetDomainInfoFromAnnotations(annotations map[string]string) (string, string, error) {
```

Comment:
The return values should have names here, otherwise it's difficult to know what is `string, string, error`